inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/InnerInlongManagerClient.java

```diff
@@ -144,12 +149,21 @@ public Boolean isGroupExists(String inlongGroupId) {
     /**
      * Get information of group.
      */
+    @SneakyThrows
     public InlongGroupInfo getGroupInfo(String inlongGroupId) {
         AssertUtils.notEmpty(inlongGroupId, "InlongGroupId should not be empty");
 
-        Response<InlongGroupInfo> responseBody = executeHttpCall(inlongGroupApi.getGroupInfo(inlongGroupId));
+        Response<Object> responseBody = executeHttpCall(inlongGroupApi.getGroupInfo(inlongGroupId));
         if (responseBody.isSuccess()) {
-            return responseBody.getData();
+            JSONObject groupInfoJson = JsonUtils.parseObject(
+                    JsonUtils.toJsonString(JsonUtils.toJsonString(responseBody.getData())),
```

Comment:
Why call `toJsonString` 2 times here?

Reply:
body is hashMap
first ( toJsonString ) make it string
second( toJsonString ) make it json_string

Reply:
See this:

<img width="1472" alt="2be5da2d56a97360df8d0608dc430e50" src="https://user-images.githubusercontent.com/31994335/174823812-5c764c72-544a-4fbb-932a-746d6fb71f32.png">